recipyCommon/utils.py

```diff
@@ -93,6 +94,21 @@ def f(self, wrapped, instance, args, kwargs):
     return f
 
 
+def create_argument_wrapper(log_input_function, log_output_function, arg_loc,
```

Comment:
Could you add a docstring or something here? I'm 95% sure I know exactly what you're doing, but it'd be good to be sure.

Reply:
Documentation added.